sale_timesheet_force_non_billable/models/account_analytic_line.py

```diff
@@ -2,21 +2,28 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from odoo import api, fields, models
+from odoo.osv import expression
 
 
 class AccountAnalyticLine(models.Model):
-    _inherit = 'account.analytic.line'
+    _inherit = "account.analytic.line"
 
     force_non_billable = fields.Boolean(
-        default=False,
-        track_visibility='onchange'
+        default=False, track_visibility="onchange"
     )
 
-    @api.multi
-    @api.depends('so_line.product_id', 'project_id', 'task_id',
-                 'force_non_billable')
+    @api.depends(
+        "so_line.product_id", "project_id", "task_id", "force_non_billable"
+    )
     def _compute_timesheet_invoice_type(self):
         timesheets = self.filtered(lambda t: t.force_non_billable)
-        timesheets.update({'timesheet_invoice_type': 'non_billable'})
-        return super(AccountAnalyticLine, self - timesheets).\
-            _compute_timesheet_invoice_type()
+        timesheets.update({"timesheet_invoice_type": "non_billable"})
+        return super(
+            AccountAnalyticLine, self - timesheets
+        )._compute_timesheet_invoice_type()
+
+    def _timesheet_get_portal_domain(self):
+        domain = super(AccountAnalyticLine,
+                       self)._timesheet_get_portal_domain()
```

Comment:
```suggestion
        domain = super()._timesheet_get_portal_domain()
```
This is the same and more readable